docs/release_notes/feb2023.md

```diff
@@ -17,41 +17,43 @@ tags:
 
 #### No team required on sign up
 
-Previously, A user was asked to enter a unique team when signing up for Prophecy. To improve the sign-up experience, requirement of providing a Team is now removed.  
-Now the user can just sign up by giving his Name, Email and Company name. He can be added to a team by Admin post sign up.
+Previously, A user was asked to enter a unique team when signing up for Prophecy. To improve the sign-up experience, this requirement has been removed
+Now the user can just sign up by giving their Name, Email and Company name. They can be added to a team by Admin after signing up.
 
 ![New_Sign_up](img/new_sign_up.png)
 
 #### Sorting on tables
 
-All metadata tables will now have options to sort on the columns. By default, the entities in table will be sorted on Modified at or Created At time in decreasing order.
+All metadata tables will now have options to sort on the columns. By default, the entities in table will be sorted on `Modified At` or `Created At` time in decreasing order.
 
 ![Sorting](img/Sorting.gif)
 
 #### Use a Cluster in Current tab
 
-A User editing Prophecy Pipelines in multiple tabs, can now detach the cluster Attached to a Pipeline in another tab, and attach to current Pipeline by clicking the same cluster.
-He will see a warning as below before the cluster is detached in previous tab.
+A User editing Prophecy Pipelines in multiple tabs can now detach the cluster Attached to a Pipeline in another tab and attach the current Pipeline by clicking the same cluster.
+They will see this warning before the cluster is detached in previous tab.
 ![Attach-cluster](img/Attach-cluster1.png)
 
-If there were running Jobs in another tab, he would be prompted to cancel the existing Jobs before switching.
+If there were running Jobs in another tab, they will be prompted to cancel the existing Jobs before switching.
 ![Attach-cluster-warning](img/Attach_cluster_job_warning.png)
 
 #### Link to Databricks Workflow in Released Prophecy Jobs
 
-In Job editor view, Added the link to the corresponding Databricks Workflow. Please note, this will only come if the Job is enabled and released.
+In Job editor view we've added the link to the corresponding Databricks Workflow.
+
```

Comment:
[slyons](https://github.com/slyons) [yesterday](https://github.com/SimpleDataLabsInc/prophecy-docs/pull/104#discussion_r1122111704)

"In Job editor view, Added" -> "**In the Job editor we've added**"